reference/orders.v2.oas2.yml

```diff
@@ -99,6 +100,8 @@ paths:
 
         After the update, the PUT request clears all discounts and promotions applied to the changed order line items. Since the order data syncs with other ERP systems, like Amazon or eBay, the updated order returns to the default setting, removing any applied discounts.
         
+        To update the fees for an order, include the fee `id` in the request body. The body should include all relevant fields, with either the updated values or the existing values you wish to retain. Fees not included in the payload will be deleted. Fees provided in the request will be updated, and fees without an ID will be created as new.
```

Comment:
It's probably worth mentioning something along the lines of 


Most sub-resources like `products`  included in the `/v2/orderds` PUT request body act more like PATCH in that they retain the existing values and only update those included in the request body. 

Fees has more standard PUT semantics in that the fees in the request body will completely replace any existing fees on the order. 
To retain an existing fee, include it in the PUT request body with it's associated `id` etc.